### ISSUE TITLE
Intermittent issues when page dynamically updates title

### ISSUE THREAD

**Issue by NordMagnus** [sylouuu/chrome-tab-modifier#168]
_In case of bug report, please fill this template and remove italic lines_
_Otherwise, clear all and type :)_

### Tab Modifier version

**Browser:** `Chrome`
**Extension version:** `0.22.0`

### Expected behavior
The rule to be applied

### Actual behavior
Showing e.g. @1 instead

### Steps to reproduce the behavior
Create a rule in e.g. JIRA and do some updates. Sometimes the placeholder (@1) is shown instead of the rule output

![image](https://user-images.githubusercontent.com/36410135/47486005-a7874700-d83f-11e8-91d3-2599c5889715.png)

![image](https://user-images.githubusercontent.com/36410135/47486155-fc2ac200-d83f-11e8-9175-b2a8e540da9e.png)

**Comment by sylouuu** [comment]
Hello,

@1 is showed when the regexp didn't match anything.

So the fallback is the title you have set.

Best

**Comment by NordMagnus** [comment]
Not sure if what you're saying is that it's not a bug or not 🙄 

Problem is that there is a matching title. Reloading the page shows it. Also it only happens when page dynamically changes title. So I'd call it a bug.

It would be better perhaps to have original tab title as fallback if no match found?

**Comment by sylouuu** [comment]
SPAs that are changing the tilte in JS after the page load, are not supported by the extension.

It's an improvement.

Best

**Comment by MartinLichtblau** [comment]
This issue seems similar to #53 
In todays time more websites are update their content dynamically which leads to some issues. Running this extension directly once the site is loaded doesn't work in some cases.

1st issue: DOM loads later 
I fixed that one by running it once onUpdated=='complete', which is already an improvements. Plus checking the DOM until the specified selector is found, trying again up to 10 times every 2 seconds.

2nd issue: DOM loads dynamically
Fixed it for many pages already by running it onUpdated=='complete', since this event fires if the url changes in any way.
If the dynamic updating doesn't change anything but the dom, it's harder. The new MutationObserver works, but is said to be resource hungry, but could be made more efficient.... but still.

In sum: run onUpdated=='complete' and poll for some time.
Could I create a PullRequest for that?
